src/uu/mv/src/mv.rs

```diff
@@ -420,7 +420,7 @@ fn rename(
             OverwriteMode::NoClobber => return Ok(()),
             OverwriteMode::Interactive => {
                 if !prompt_yes!("overwrite {}?", to.quote()) {
-                    return Ok(());
+                    return Err(io::Error::new(io::ErrorKind::Other, ""));
```

Comment:
Can we do something else than constructing an `io::Error`? This is going to give a weird error message isn't it? For a PR like this, I guess it's OK, though.

Reply:
well, there isn't any error message :)
```
$ touch a b && echo "n"|./target/debug/coreutils mv -i a b; echo $?
mv: overwrite 'b'? 
1
```